server.js

```diff
@@ -3,11 +3,6 @@ global.rootRequire = function(name) {
   return require(__dirname + '/' + name);
 }
 
-// Nodejitsu specifies NODE_ENV='production' by default
```

Comment:
This may help explain the 2 New Relic integrations we're seeing in https://github.com/DoSomething/ds-mdata-responder/issues/518#issuecomment-233755953.